src/hub/Hub.sol

```diff
@@ -101,7 +101,7 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
     //----------------------------------------------------------------------------------------------
 
     /// @inheritdoc IHub
```

Comment:
Just noticed I accidentally changed the section header above this to FreezeOnly methods, should be Permissionless methods. Could you fix?

Reply:
Done